hazelcast/src/main/java/com/hazelcast/instance/NodeExtension.java

```diff
@@ -130,7 +131,7 @@
     /**
      * Creates factory method that creates server side client message handlers
      */
-    MessageTaskFactory createMessageTaskFactory();
+    List<? extends MessageTaskFactory> createMessageTaskFactories();
```

Comment:
The Javadoc doesn't match the method name AFAIK. And why is a list needed? Is ordering relevant?